components/external-services-mock/internal/notification/handler_test.go

```diff
@@ -63,7 +63,66 @@ func TestHandler_Patch(t *testing.T) {
 			r := httptest.NewRecorder()
 
 			//WHEN
-			h.Patch(r, req)
+			patchHandler := h.Patch
+			patchHandler(r, req)
```

Comment:
why do we need this? couldn't we use `h.Patch(r, req)`?
P.S. Valid for the other places as well